dev-manuals/test-plans/minigames/git-card-game-frontend/plans/u-git-card-game-1-game-start.md

```diff
@@ -0,0 +1,32 @@
+# Git Card Game start (`u.git-card-game-1`)
+
+Version: V1.1, 11.08.2022, Created test plan \
```

Comment:
```suggestion
Version: V1.0, 11.08.2022, Created test plan \
```
isnt this version 1.0 if you just created it?

Reply:
It will be 1.1 after my fixes